src/test/java/LinkedBinarySearchTreeEqualsTest.java

```diff
@@ -0,0 +1,76 @@
+import com.google.common.testing.EqualsTester;
+import org.junit.jupiter.api.Test;
+
+public class LinkedBinarySearchTreeEqualsTest {
+
+    @Test
+    @SuppressWarnings("UnstableApiUsage")
+    public void equals_verify_contract() {
```

Comment:
There are no groups testing `BinaryTree<T>` equality. Really, one should be part of each group defined here.

Though, I don't recall if we have any other tree implementations. So. :shrug:

Reply:
You're right --- there are no other binary tree implementations.